samples/SampleOrigin/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.AspNetCore.Hosting;
+
+namespace SampleOrigin
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var host = new WebHostBuilder()
+                .UseKestrel()
+                .UseUrls("http://*:5001")
+                .UseContentRoot(Directory.GetCurrentDirectory())
```

Comment:
Hmm, it's actually not exactly right to use `GetCurrentDirectory()` because the current directory might not be the directory that contains the EXE. E.g. if I run the app by doing `..\another_folder\foo.exe` then the "current directory" is where "I am at" and not "where the executable is at".

Is there a better way to find the "directory we want"?